creepy/protocol/handshake.py

```diff
@@ -32,27 +32,36 @@ class HandshakeProtocol:
 
     def __init__(self, authorized_keys_path=None):
         self.salt = secrets.token_bytes(self.SALT_SIZE)
+        bobs = self._add_keys(authorized_keys_path)
+        self._bobs = bobs
+
+    def _add_key(self, key, bobs):
```

Comment:
```suggestion
    def _add_bob_key(self, key):
```
and make it add directly to `self._bobs`.